DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -29,45 +48,40 @@
 
-### Additional Steps
-- Make a copy of "build.properties.example" contained in "sormas-base", rename it to "build.properties" and set "glassfish.domain.root" to the location of the sormas domain located in the "glassfish/domains" folder inside your payara installation
-- Drag the "build.xml" file contained in "sormas-base" into the Ant view in Eclipse
-  - Either: Run "Maven install" on the sormas-base project
-  - Or: Execute the "install [default]" ant script (this needs a maven installation on your system with the M2_HOME variable set)
-  - Then: Execute the "deploy-serverlibs" ant script
-- Highlight all Eclipse projects and choose "Maven -> Update Project" from the right click menu; perform the update for all projects
-- Start the Glassfish server and deploy "sormas-ear", "sormas-rest" and "sormas-ui" by dragging the respective projects onto it, or use the "Add and Remove..."-function by right clicking on the server.
-- Open your browser and type in "http://localhost:6080/sormas-ui" or "https://localhost:6081/sormas-ui" to test whether everything has been set up correctly (and to use the application)
-
 ## IntelliJ
 - Install the latest Ultimate edition IntelliJ
 - Set the project SDK to use the installed Zulu Java 8 SDK
 - Clone the SORMAS-Project repository and open the project in IntelliJ
 	- make sure the under "File -> Project Structure -> Modules" all the modules (except the android app - this should not be added) are recognized, if not add the modules with +
-- Make sure under "File -> Settings -> Plugins" Glassfish & Ant integrations are enabled (look into the "Installed" tab)
+- Make sure under "File -> Settings -> Plugins" Glassfish integration is enabled (look into the "Installed" tab)
 - Install the Vaadin Designer plugin
-- Make a copy of "build.properties.example" contained in "sormas-base", rename it to "build.properties" and set "glassfish.domain.root" to the location of the sormas domain located in the "glassfish/domains" folder inside your payara installation
-- Run "Maven install" on the sormas-base project
 - Add a Payara server to IntelliJ:
+    - if you've chosen the setup based on [SERVER_SETUP](SERVER_SETUP.md), you'll have to adapt the following paths accordingly
 	- go to "Run -> Edit configurations"
-	- add new configuration and choose from the templates Glassfish server
-	- select the payara5 directory for application server - and name the application server field Payara5
-	- specify server domain and credentials from the server setup
+	- add **new configuration** and choose from the templates **"Glassfish Server local"**
+	    - **Configure** the payara5 directory for application server from the cargo with **Glassfish Home**: `sormas-cargoserver/target/cargo/installs/payara-5.194/payara5` (find the current version in [pom.xml](sormas-cargoserver/pom.xml))
```

Comment:
**Configure** the payara5 directory with **Glassfish Home**: `sormas-cargoserver/target/cargo/installs/payara-5.194/payara5` (find the current version in [pom.xml](sormas-cargoserver/pom.xml) **or** `/opt/payara5`